Makefile

```
@@ -0,0 +1,72 @@
+# make build MARKET=rustore
+
+# 3️⃣ make show-market
+# 4️⃣ help make
+
+SHELL := /bin/bash
+MARKET ?= google-play
+
+# Target file that contains the market link used on Android
+FILE := lib/config/constants/app_constants.dart
+
+.PHONY: show-market set-google-play set-rustore build-google-play build-rustore build help
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)